patches/api/0400-Added-PermissibleBase-getter-and-setter.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Badbird5907 <50347938+Badbird5907@users.noreply.github.com>
+Date: Wed, 12 Oct 2022 11:30:20 -0400
+Subject: [PATCH] Added PermissibleBase getter and setter
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 1d98abff1ad0116f7a2599f078aa730cb84843c1..e5db215a400a7be9fae7fd13c6d4040f3bece296 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -2832,4 +2832,21 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     @Override
+     Spigot spigot();
+     // Spigot end
++
++    // Paper start
++    /**
++     * Set the player's {@link org.bukkit.permissions.PermissibleBase}
++     * @param base {@link org.bukkit.permissions.PermissibleBase}
++     * @return the old {@link org.bukkit.permissions.PermissibleBase}
```

Comment:
These links seems a little bit too redundant, maybe keep only the links into the description but not into the parameter explanation ?